openssl-dynamic/src/main/java/io/netty/tcnative/jni/Buffer.java

```diff
@@ -15,22 +30,11 @@
  *  limitations under the License.
  */
 
-package org.apache.tomcat.jni;
+package io.netty.tcnative.jni;
 
 import java.nio.ByteBuffer;
 
-/** Buffer
- *
- * @author Mladen Turk
- */
-public class Buffer {
-    /**
-     * Deallocates or frees a memory block used by ByteBuffer
-     * <br><b>Warning :</b> Call this method only on ByteBuffers
-     * that were created by calling Buffer.alloc or Buffer.calloc.
-     * @param buf Previously allocated ByteBuffer to be freed.
-     */
-    public static native void free(ByteBuffer buf);
+public final class Buffer {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1118)